google-account-plugin/src/com/google/gct/login/GoogleLogin.java

```diff
@@ -640,7 +656,14 @@ public boolean askYesOrNo(String title, String message) {
       String updatedMessage = message;
       if (message.equals("Are you sure you want to sign out?")) {
```

Comment:
not related to your change, but this code is crazy. (just saying. I'm not asking you to fix it)